### ISSUE TITLE
WildFlyGelfLogHandler does not honor "enabled" attribute of <custom-handler>

### ISSUE THREAD

**Issue by HonoluluHenk** [mp911de/logstash-gelf#170]
Testcase
* Install the wildfly module as described in the readme: https://github.com/mp911de/logstash-gelf#wildfly
* Configure standalone.xml (wildfly version does not seem to matter):
```xml
<custom-handler name="Some-Graylog"
                            class="biz.paluch.logging.gelf.wildfly.WildFlyGelfLogHandler"
                            module="biz.paluch.logging"
                            enabled="false">
  <level name="INFO" />
  <!-- other settings do not matter -->
</custom-handler>
```

Even though the attribute "enabled" is set to "false", I see logs if there's nothing listening on localhost:12201:
LogManager error of type GENERIC_FAILURE: Connection refused
java.io.IOException: Cannot send data to localhost:12200

A first glance into the WildFlyGelfLogHandler.java Source does not show anything about reading the "enabled" attribute

Full Error stack:

```
19:23:56,490 INFO  [org.jboss.as.connector] (MSC service thread 1-5) WFLYJCA0009: Starting JCA Subsystem (WildFly/IronJacamar 1.4.6.Final)
LogManager error of type GENERIC_FAILURE: Connection refused
java.io.IOException: Cannot send data to localhost:12200
	at biz.paluch.logging.gelf.intern.sender.GelfTCPSender.sendMessage(GelfTCPSender.java:126)
	at biz.paluch.logging.gelf.jul.GelfLogHandler.publish(GelfLogHandler.java:134)
	at biz.paluch.logging.gelf.wildfly.WildFlyGelfLogHandler.publish(WildFlyGelfLogHandler.java:83)
	at org.jboss.logmanager.LoggerNode.publish(LoggerNode.java:314)
	at org.jboss.logmanager.LoggerNode.publish(LoggerNode.java:322)
	at org.jboss.logmanager.LoggerNode.publish(LoggerNode.java:322)
	at org.jboss.logmanager.LoggerNode.publish(LoggerNode.java:322)
	at org.jboss.logmanager.LoggerNode.publish(LoggerNode.java:322)
	at org.jboss.logmanager.Logger.logRaw(Logger.java:850)
	at org.jboss.logmanager.Logger.log(Logger.java:802)
	at org.jboss.logging.JBossLogManagerLogger.doLogf(JBossLogManagerLogger.java:53)
	at org.jboss.logging.Logger.logf(Logger.java:2398)
	at org.jboss.as.jaxrs.logging.JaxrsLogger_$logger.resteasyVersion(JaxrsLogger_$logger.java:129)
	at org.jboss.as.jaxrs.JaxrsSubsystemAdd.performBoottime(JaxrsSubsystemAdd.java:60)
	at org.jboss.as.controller.AbstractBoottimeAddStepHandler.performRuntime(AbstractBoottimeAddStepHandler.java:116)
	at org.jboss.as.controller.AbstractAddStepHandler$1.execute(AbstractAddStepHandler.java:151)
	at org.jboss.as.controller.AbstractOperationContext.executeStep(AbstractOperationContext.java:980)
	at org.jboss.as.controller.AbstractOperationContext.processStages(AbstractOperationContext.java:726)
	at org.jboss.as.controller.AbstractOperationContext.executeOperation(AbstractOperationContext.java:450)
	at org.jboss.as.controller.ParallelBootOperationStepHandler$ParallelBootTask.run(ParallelBootOperationStepHandler.java:386)
19:23:56,427 INFO  [org.wildfly.extension.io] (ServerService Thread Pool -- 35) WFLYIO001: Worker 'default' has auto-configured to 8 core threads with 64 task threads based on your 4 available processors
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
19:23:56,480 INFO  [org.jboss.as.security] (ServerService Thread Pool -- 49) WFLYSEC0002: Activating Security Subsystem
	at org.jboss.threads.JBossThread.run(JBossThread.java:320)
Caused by: java.net.ConnectException: Connection refused
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at biz.paluch.logging.gelf.intern.sender.GelfTCPSender.connect(GelfTCPSender.java:164)
	at biz.paluch.logging.gelf.intern.sender.GelfTCPSender.sendMessage(GelfTCPSender.java:103)
	... 23 more
```

**Comment by HonoluluHenk** [comment]
I created a pull request with a fix: 13c29ae6c6e39509c544e114b4a852c0396385c5

**Comment by mp911de** [comment]
Thanks a lot. Solved via #171.
